storage/Cargo.toml

```diff
@@ -16,6 +16,8 @@ hex = "0.4.3"
 hmac = { version = "0.12.1", optional = true }
 http = { version = "0.2.8", optional = true }
 httpdate = { version = "1.0", optional = true }
+hyper = {version = "0.14.11", optional =  true}
```

Comment:
Is this necessary? Is `reqwest` haven't supported unix domain socket?

Reply:
Yes. `reqwest` has not supported unix socket yet. The issue https://github.com/seanmonstar/reqwest/issues/39 is still open.